src/api/fetchers.ts

```diff
@@ -18,6 +18,8 @@ import type {
   LocalLoginResponse,
   ApiGetFieldInfoResponse,
   SearchableSpecificField,
+  ApiListDrawersResponse as ApiListDrawersResponse,
```

Comment:
I don't understand this line ... what's it doing?

Reply:
Hah – This was probably generated by a VSCode rename. I'll fix it.